### PR TITLE
improve: support specifying session in the sync symbol

### PR DESCRIPTION
**Pull request by c9s**

### CODE REVIEW

File: pkg/bbgo/config.go [hunk 1]
```diff
@@ -208,12 +208,53 @@ func GetNativeBuildTargetConfig() BuildTargetConfig {
...
+		}
+	}
+	return m, rest
+}
+
 type SyncConfig struct {
 	// Sessions to sync, if ignored, all defined sessions will sync
 	Sessions []string `json:"sessions,omitempty" yaml:"sessions,omitempty"`
 
 	// Symbols is the list of symbol to sync, if ignored, symbols wlll be discovered by your existing crypto balances
```
Comment (zenixls2) [inline pkg/bbgo/config.go:256]: the list of {symbol, session} pairs to sync

Comment (c9s) [reply pkg/bbgo/config.go:256]: 👍

File: pkg/cmd/sync.go [hunk 2]
```diff
@@ -89,7 +89,9 @@ var SyncCmd = &cobra.Command{
 
 		if len(symbol) > 0 {
 			if userConfig.Sync != nil && len(userConfig.Sync.Symbols) > 0 {
-				userConfig.Sync.Symbols = []string{symbol}
+				userConfig.Sync.Symbols = []bbgo.SyncSymbol{
+					{Symbol: symbol},
```
Comment (zenixls2) [inline pkg/cmd/sync.go:93]: what about sessionNames?

Comment (c9s) [reply pkg/cmd/sync.go:93]: session name is optional, when session name is empty, the symbol will be applied to all sessions.

**Pull request merged**